### PR TITLE
Closed #105 - Fix fonts IOS

### LINKED ISSUE #105: Fix fonts IOS

### PR DESCRIPTION
**Pull request by strelovidniy**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [x] @github_username

### Second Level Review

- [x] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

**Review by IhorSolovii — APPROVED** [review]

File: SoftServeCarMobile/package.json [hunk 1]
```diff
@@ -80,7 +80,7 @@
     "jest": "25.1.0",
     "jest-mock-axios": "4.2.1",
     "metro-react-native-babel-preset": "0.59.0",
-    "react-native-linear-gradient": "2.5.6",
+    "react-native-linear-gradient": "^2.5.6",
```
Comment (Reennon) [inline SoftServeCarMobile/package.json:83]: remove '^'

File: SoftServeCarMobile/yarn.lock [hunk 2]
```diff
@@ -9,7 +9,7 @@
   dependencies:
     "@babel/highlight" "^7.10.4"
 
-"@babel/core@^7.0.0", "@babel/core@^7.0.0-0", "@babel/core@^7.1.0", "@babel/core@^7.7.5", "@babel/core@7.8.4":
+"@babel/core@^7.0.0", "@babel/core@^7.1.0", "@babel/core@^7.7.5", "@babel/core@7.8.4":
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:12]: remove '^'

File: SoftServeCarMobile/yarn.lock [hunk 3]
```diff
@@ -356,7 +356,7 @@
   dependencies:
     "@babel/helper-plugin-utils" "^7.10.4"
 
-"@babel/plugin-syntax-flow@^7.0.0", "@babel/plugin-syntax-flow@^7.12.13", "@babel/plugin-syntax-flow@^7.2.0":
+"@babel/plugin-syntax-flow@^7.0.0", "@babel/plugin-syntax-flow@^7.12.1", "@babel/plugin-syntax-flow@^7.2.0":
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:359]: remove '^'

File: SoftServeCarMobile/yarn.lock [hunk 4]
```diff
@@ -499,13 +499,13 @@
...
-  "version" "7.12.13"
+"@babel/plugin-transform-flow-strip-types@^7.0.0", "@babel/plugin-transform-flow-strip-types@7.12.10":
+  "integrity" "sha512-0ti12wLTLeUIzu9U7kjqIn4MyOL7+Wibc7avsHhj4o1l5C0ATs8p2IMHrVYjm9t9wzhfEO6S3kxax0Rpdo8LTg=="
+  "resolved" "https://registry.npmjs.org/@babel/plugin-transform-flow-strip-types/-/plugin-transform-flow-strip-types-7.12.10.tgz"
+  "version" "7.12.10"
   dependencies:
-    "@babel/helper-plugin-utils" "^7.12.13"
-    "@babel/plugin-syntax-flow" "^7.12.13"
+    "@babel/helper-plugin-utils" "^7.10.4"
+    "@babel/plugin-syntax-flow" "^7.12.1"
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:508]: remove '^'

File: SoftServeCarMobile/yarn.lock [hunk 5]
```diff
@@ -499,13 +499,13 @@
...
-  "resolved" "https://registry.npmjs.org/@babel/plugin-transform-flow-strip-types/-/plugin-transform-flow-strip-types-7.12.13.tgz"
-  "version" "7.12.13"
+"@babel/plugin-transform-flow-strip-types@^7.0.0", "@babel/plugin-transform-flow-strip-types@7.12.10":
+  "integrity" "sha512-0ti12wLTLeUIzu9U7kjqIn4MyOL7+Wibc7avsHhj4o1l5C0ATs8p2IMHrVYjm9t9wzhfEO6S3kxax0Rpdo8LTg=="
+  "resolved" "https://registry.npmjs.org/@babel/plugin-transform-flow-strip-types/-/plugin-transform-flow-strip-types-7.12.10.tgz"
+  "version" "7.12.10"
   dependencies:
-    "@babel/helper-plugin-utils" "^7.12.13"
-    "@babel/plugin-syntax-flow" "^7.12.13"
+    "@babel/helper-plugin-utils" "^7.10.4"
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:507]: also remove '^'

File: SoftServeCarMobile/yarn.lock [hunk 6]
```diff
@@ -688,7 +688,7 @@
     "core-js-pure" "^3.0.0"
     "regenerator-runtime" "^0.13.4"
 
-"@babel/runtime@*", "@babel/runtime@^7.0.0", "@babel/runtime@^7.4.4", "@babel/runtime@^7.6.0", "@babel/runtime@^7.8.4", "@babel/runtime@7.8.4":
+"@babel/runtime@^7.0.0", "@babel/runtime@^7.4.4", "@babel/runtime@^7.6.0", "@babel/runtime@^7.8.4", "@babel/runtime@7.8.4":
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:691]: - '^'

File: SoftServeCarMobile/yarn.lock [hunk 7]
```diff
@@ -7034,7 +7034,7 @@
     "prop-types" "^15.7.2"
     "yargs" "^15.0.2"
 
-"react-native@*", "react-native@^0.63.4", "react-native@>=0.46":
+"react-native@^0.63.4":
```
Comment (Reennon) [inline SoftServeCarMobile/yarn.lock:7037]: remove '^'

**Review by Reennon — DISMISSED** [review]
Minor changes request

**Review by RamzesUAA — APPROVED** [review]

**Pull request merged**